package-lock.json

```diff
@@ -58298,7 +58298,7 @@
         "@leafygreen-ui/hooks": "^7.3.3",
         "@leafygreen-ui/icon": "^11.21.0",
         "@leafygreen-ui/icon-button": "^15.0.3",
-        "@leafygreen-ui/info-sprinkle": "*",
+        "@leafygreen-ui/info-sprinkle": "^1.0.0",
```

Comment:
Is this change on purpose?

Reply:
`npm run bootstrap` made this change. as CI passes, I think its fine